test/commentLikes.spec.js

```diff
@@ -37,15 +37,15 @@ describe('User can like a comment test', () => {
           done();
         });
     });
-    it('should unlike a comment', (done) => {
+    it('should resetReaction a comment', (done) => {
```

Comment:
change _resetReaction a comment_ to _remove reaction on a comment_ . This will make your test description clear.